_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+One of the external factors is the USD -> CDN conversion rate, currency fluctuations directly influence our purchase prices. It probably explains why the cost per TB on an individual disk was higher in September 2016. Our storage need at that time was only of 3 servers (864 TB), so the impact was fairly limited and the conversion rate started to go back down after that time and is today at the same level it was early 2015 (although still much higher than when the project started in 2014).
+
+<center>
+  <figure style="width: 70%;">
+      <img src="{{site.urlimg}}francois_gerthoffert/collab-procurement/usd-cdn-conversion-rate.png"/>
+      <figcaption>USD-CDN exchange rate evolution</figcaption>
+  </figure>
+</center>
+
+But ultimately, the thing to look at is that today we are purchasing servers with 2.5 times more storage capacity than in 2015, which means a need for less physical resources per TB of storage, which plays largely in our favor from a budget point of view.
+
+# Conclusion
+
+So what could be the recommendations and conclusion of this article ?
+
+If you know upfront that your project will be composed of multiple purchases, identify a procurement strategy and setup a contractual framework before starting. Not having to re-negotiate or re-validate items such as warranty, support details, greatly eases the procurement process.
+
+Don't purchase more storage capacity than required. Considering that transferring petabytes of data takes a large amount of time, you might have sufficient time for staggered purchases. Instead, identify your requirements and schedule your purchases accordingly. This will provide the project with better value per dollar spent.
+
+Watch out for external factors, such as currency exchange rate. Even if they are outside of your control, you might want to try to articulate your strategy around those.
+
+Establish a clear, fair and re-usable evaluation matrix. This will facilitate vendor evaluation and could also be of a great help may a vendor decide to contest an award. This is particularly true in the public sector. Furthermore, in the context of a particular purchase, don't talk to vendors directly before a quote is awarded, let your procurement department handle the communication. This will help prevent disputes coming from vendors. IT is a relatively small ecosystem and it's not rare from vendors to talk to each other or hear rumors and the last thing you want is to have to re-run a quotation process.
+
+Finally, don't forget that you are not on your own in the process. Your procurement department have experience running such purchases and although they might not have experience in your particular technical domain, they will be able to walk you through the procurement process, help catch potential errors in your request and ensure communication with vendors follow procurement rules and practices.
```

Comment:
Finally, don't forget that you are not on your own in the process. Your procurement department has experience running such purchases and although they might not have experience in your particular technical domain, they will be able to walk you through the procurement process, help catch potential errors in your request and ensure communication with vendors follows procurement rules and practices.